### PR TITLE
Feature/se 2412.adjust holdings example

### PR DESCRIPTION
**Pull request by ryangarvey**

### CODE REVIEW

**Comment by slemasne** [comment]
Good works.

But lets have 5 files:

```
1-create-transaction-portfolio.sql
2-create-instruments.sql
3-set-holdings.sql
4-adjust-holdings.sql
5-cancel-holdings.sql
```

We number the files with a prefix so they run in order.

File: examples/lusid/holdings/3-create-holdings.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+use Drive.Excel with @@file_date
+--file=/luminesce-examples/equity_holdings_{@@file_date}.xlsx
+--worksheet=lusid_holdings
+--addFileName
+enduse;
+
+-- Run holdings transformation
+@holdings_for_upload =
+
+select @@portfolioCode1 as [PortfolioCode], @@portfolioScope as [PortfolioScope], holding_date as
```
Comment (slemasne) [inline examples/lusid/holdings/3-create-holdings.sql:33]: Is this a **set** or **adjust** by default? 
Add the WriteAction parameter which explicitly sets the holding?

File: examples/lusid/holdings/1-create-transaction-portfolio.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
+-- ===============================================================
+-- Description:
+-- 1. In this query, we create a Transaction Portfolio in LUSID
+-- ===============================================================
+
+-- Defining scope and code variables
+@@portfolioScope =
+
+select 'LuminesceHoldingsExample';
```
Comment (slemasne) [inline examples/lusid/holdings/1-create-transaction-portfolio.sql:9]: Change all scopes to `luminesce-examples`
Just to keep it consistent.

File: examples/lusid/holdings/5-cancel-holdings.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        @@portfolioCode1 as PortfolioCode,
+        #2022-04-21# as EffectiveAt,
+        'LUID_00003D6Z' as LusidInstrumentId,
+        'Cancel' as WriteAction
+
+        union all 
+
+select  @@portfolioScope as PortfolioScope,
+        @@portfolioCode1 as PortfolioCode,
+        #2022-04-21# as EffectiveAt,
```
Comment (slemasne) [inline examples/lusid/holdings/5-cancel-holdings.sql:28]: Remove this if not needed?

File: examples/lusid/holdings/3-create-holdings.sql (hunk 1 above)
Comment (ryangarvey) [reply examples/lusid/holdings/3-create-holdings.sql]: This file has been removed and as the new `3-set-holdings.sql` performs identically just with the explicit SET value.

**Review by slemasne — CHANGES_REQUESTED** [review]

File: examples/lusid/holdings/5-cancel-holdings.sql (hunk 3 above)
Comment (ryangarvey) [reply examples/lusid/holdings/5-cancel-holdings.sql:19]: Removed everything without throwing errors.

File: examples/lusid/holdings/3-set-holdings.sql [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+select 'LuminesceHoldingsExample';
+
+@@portfolioCode1 =
+
+select 'UkEquity';
+
+-- Create new table to store holding's data
+@holding_data = 
+
+select  @@portfolioScope as PortfolioScope,
```
[the remark this answers is not in the document]
Comment (ryangarvey) [reply examples/lusid/holdings/3-set-holdings.sql:32]: Holdings are now loaded in from XLSX file

File: examples/lusid/holdings/3-set-holdings.sql [hunk 5]
```diff
@@ -0,0 +1,47 @@
+-- =======================================================
+-- Description:
+-- 1. In this query, we add a new holding to the portfolio
```
Comment (slemasne) [inline examples/lusid/holdings/3-set-holdings.sql:3]: "add new holdings" 
We're adding multiple

File: examples/lusid/holdings/3-set-holdings.sql [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+select  @@portfolioScope as PortfolioScope,
+        @@portfolioCode1 as PortfolioCode,
+        holding_date as EffectiveAt,
+        InstrumentId as ClientInternal,
+        currency as CostCurrency,
+        units as Units,
+        12.3 as CostPrice,
+        #2022-04-19# as PurchaseDate,
+        #2022-04-21# as SettleDate,
```
Comment (slemasne) [inline examples/lusid/holdings/3-set-holdings.sql:40]: Can these be removed if not needed?

**Review by slemasne — APPROVED** [review]

**Pull request merged**